include/fmt/std.h

```diff
@@ -17,10 +17,16 @@
 #if FMT_HAS_INCLUDE(<version>)
 #  include <version>
 #endif
+#if FMT_CPLUSPLUS >= 201703L
```

Comment:
What is this check for? Shouldn't `FMT_HAS_INCLUDE` and feature test macro checks be enough?

Reply:
Could you add a short comment clarifying that this check is for MSVC?

Reply:
@vitaut Done